core/src/main/java/edu/wpi/grip/core/sources/IPCameraFrameGrabber.java

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
I don't see any changes in the CameraSource. How is this one being imported? Doesn't the bytdeco import take precedence?